src/views/router/router.tsx

```diff
@@ -2,21 +2,16 @@ import { useState } from 'react'
 import { Route, appRoutes } from './routes'
 
 import styles from './router.module.scss'
-import Settings from '../settings/settings'
+import Navbar from '@/components/navbar/navbar'
 
 const Router = () => {
 	const [activeTab, setActiveTab] = useState<Route>(appRoutes.Home)
 
 	return (
 		<div className={styles['wrapper']}>
-			<Settings />
-			<nav className={styles['navigation-tab--wrapper']}>
-				{Object.values(appRoutes).map((route) => (
-					<button key={route.label} onClick={() => setActiveTab(route)}>
-						{route.label}
-					</button>
-				))}
-			</nav>
+			<div className={styles['navbar']}>
+				<Navbar setActiveTab={setActiveTab} appRoutes={appRoutes} />
+			</div>
```

Comment:
Same as the router.module.scss comment. We can probably remove the div parent and just have the Navbar component handle that.